plugins/carla/cmake/carla-discovery-native.cmake

```diff
@@ -0,0 +1,35 @@
+add_executable(carla-discovery-native)
+mark_as_advanced(carla-discovery-native)
+
+set(carla_discovery_extra_libs ${carla_pthread_libs})
+
+if(OS_MACOS)
+  find_library(APPKIT AppKit)
+  mark_as_advanced(APPKIT)
+  set(carla_discovery_extra_libs ${carla_discovery_extra_libs} ${APPKIT})
```

Comment:
Use `"$<LINK_LIBRARY:FRAMEWORK,AppKit.framework>"` to link against Framework targets.

Reply:
no longer applies, marking as resolved (trick done on carla side now, thanks!)